modulefiles/gsi_jet.lua

```diff
@@ -1,24 +1,23 @@
 help([[
 ]])
 
-prepend_path("MODULEPATH", "/mnt/lfs4/HFIP/hfv3gfs/role.epic/hpc-stack/libs/intel-18.0.5.274/modulefiles/stack")
+prepend_path("MODULEPATH", "/mnt/lfs4/HFIP/hfv3gfs/role.epic/miniconda3/modulefiles")
+miniconda3_ver=os.getenv("miniconda3_ver") or "4.12.0"
+load(pathJoin("miniconda3", miniconda3_ver))
+
+prepend_path("MODULEPATH", "/mnt/lfs4/HFIP/hfv3gfs/role.epic/hpc-stack/libs/intel-2022.1.2/modulefiles/stack")
```

Comment:
Does this `prepend_path` point at the official installation all UFS and EMC apps should use?

Reply:
@natalie-perlin confirmed this is the correct HPC-stack location.

Reply:
Resolve with caveat noted above that g-w and library team staff need to confirm that machine specific modulefiles in this PR use officially maintained and supported modules.